package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Mar 12 07:18:11 UTC 2021 - Knut Anderssen <kanderssen@suse.com>
+
+- Ensure that during an Autoinstallation the network configuration
```

Comment:
What about something shorter like *AutoYaST: configure but not apply the network configuration at the end of the 1st stage*?